src/modules/common/components/TransactionSocketWrapper/index.js

```diff
@@ -0,0 +1,81 @@
+import React, { useEffect } from 'react';
+import { useQueryClient } from '@tanstack/react-query';
+import { toast } from 'react-toastify';
+import client from 'src/utils/api/client';
+import { TRANSACTIONS, AUTH } from 'src/const/queries';
+import { MODULE_COMMANDS_NAME_MAP } from '@transaction/configuration/moduleCommand';
+import { fromRawLsk } from '@token/fungible/utils/lsk';
+import i18n from 'src/utils/i18n/i18n';
+import { useCurrentAccount } from '@account/hooks';
+import { useCurrentApplication } from '@blockchainApplication/manage/hooks';
+import { useTransactions, useTransactionsConfig } from '@transaction/hooks/queries';
+
+const filterIncomingTransactions = (transactions, account) =>
+  transactions.filter(
+    (transaction) =>
+      transaction &&
+      transaction.moduleCommand === MODULE_COMMANDS_NAME_MAP.transfer &&
+      transaction.params.recipientAddress === account.metadata.address
+  );
+
+const showNotificationsForIncomingTransactions = (transactions, account, token) => {
+  filterIncomingTransactions(transactions, account).forEach((transaction) => {
+    const amount = fromRawLsk(transaction.params.amount);
+    if (amount > 0) {
+      const message = transaction.params.data
+        ? i18n.t('with message {{message}}', { message: transaction.params.data })
+        : '';
+      // eslint-disable-next-line no-new
+      new Notification(i18n.t('{{amount}} {{token}} Received', { amount, token }), {
+        body: i18n.t('Your account just received {{amount}} {{token}} {{message}}', {
+          amount,
+          token,
+          message,
+        }),
+      });
+      toast.info(
+        i18n.t('You just received {{amount}} {{token}}', {
```

Comment:
This is translation should also be pushed to common.json

Reply:
@oskarleonard Translations are automatically pushed to common.json when triggered from `react-i18n`. However, this translation is triggered inside a regular function, rather than a functional component (which is why the import is different) and hence isn't automatically pushed to common.json. So it's fine as is